lib/components/Selection control/Checkbox/checkbox.scss

```diff
@@ -0,0 +1,109 @@
+@import "../../../styles/index.scss";
```

Comment:
`index.scss` is imported in all css files by default. Can you check if we can remove this?